### ISSUE TITLE
Custom buttons need to show options for whom an event should be triggered

### ISSUE THREAD

**Issue by maddalihanumateja** [maddalihanumateja/ParticipateInRemoteDesign#2]
Maybe this would look like a drop down box with the names of the meeting participants. It can appear next to the custom button when clicked. Selecting one of the names should trigger an event for that participant alone (maybe later it can trigger different kinds of events for different participants). For example, clicking on the print button opens a drop down box with the names ['anon1','anon2',..]. Clicking on anon1 and selecting a file to send will activate anon1's printer. This should also, probably take into account the kinds of devices that anon1 has. If anon1 doesn't have a printer then their name shouldn't appear on the dropdown box for the print button, This can be in a different issue (created a new issue #3 ).

**Comment by ameliashort331** [comment]
Right now, I have a custom button at the footer labeled "Devices":
![image](https://user-images.githubusercontent.com/13869404/98396774-43054200-202c-11eb-8044-4a5f876864d2.png)
![image](https://user-images.githubusercontent.com/13869404/98396787-47c9f600-202c-11eb-8639-61f52284c62d.png)

Assume that the participant's name is Jane, and Jane has a printer and projector next to her. If a researcher clicks on the devices button, it will bring up a modal with available devices possible actions to take:
![image](https://user-images.githubusercontent.com/13869404/98397021-b14a0480-202c-11eb-9bfa-05226818126b.png)

There is also a select list that is dynamically populated when participants enter the meeting:
![image](https://user-images.githubusercontent.com/13869404/98397078-ca52b580-202c-11eb-9930-30c81ca8ffcc.png)

In this scenario, the researcher will send files from the modal, rather than the "manage participants" tab, as demonstrated in [issue #39
](https://github.com/maddalihanumateja/ParticipateInRemoteDesign/issues/39)

This is just a tentative design for now. Would love to gather some feedback on whether or not having a dynamic modal makes the most sense for the researchers' anticipated tasks.

**Comment by maddalihanumateja** [comment]
I like the visual aesthetic of this. Just wondering if having the modal appear towards the side instead of obstructing the video of the participant is more practical.  

Does the select list actually check if the device is available at the user's location before it is dynamically populated?

Also, related to a previous issue, are printer and projector hard coded on the modal? Or do we have a list of device to search for? (So if there's a camera added to the system, we don't need to manually add the camera to the modal). There's a lot of blank space on the modal, so I assumed you reserved those for future devices. This is also a point you don't need to address immediately.

**Comment by ameliashort331** [comment]
tldr; it's all hard coded right now, just to show what the modal could look like. Noted about its placement on the screen!

I've been thinking about this issue a lot in conjunction with [issue #43](https://github.com/maddalihanumateja/ParticipateInRemoteDesign/issues/43), and just generally what's the best way to get the Raspberry Pi set up to the wifi. The more I think about it, I think that the Bluetooth approach and scheduling a short logistics session might be better for this project.

With the Bluetooth approach, there is already code written to send device information to the Raspberry Pi. It wouldn't be very difficult to then take that information and send it to the webapp. The webapp can then use this response to easily dynamically populate elements on the modal. More discussion about the Bluetooth approach are posted on the [relevant issue](https://github.com/maddalihanumateja/ParticipateInRemoteDesign/issues/43).

If we like this approach, then this can be something I work on over the next week (Nov. 6- Nov. 13)
